backend/plugins/sales_api/src/modules/sales/db/models/Stages.ts

```
@@ -0,0 +1,97 @@
+      if (doc.code) {
+        await this.checkCodeDuplication(doc.code);
+      }
+      return models.Stages.create(doc);
+    }
+
+    /**
+     * Update Stage
+     */
+    public static async updateStage(_id: string, doc: IStage) {
+      if (doc.code) {
+        await this.checkCodeDuplication(doc.code);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)